src/pudl/output/ferc1.py

```diff
@@ -228,6 +228,55 @@ def _out_ferc1__yearly_plants_utilities(
     )
 
 
+## example/draft factory pattern
+specs = [
+    {
+        "name": "out_ferc1__yearly_purchased_power_and_exchanges_sched326",
```

Comment:
Note that in all cases, there's an easy way to construct the input (core) and output asset names, given just a single string, (`purchased_power_and_exchanges_sched326` in this case) -- they all have the same prefix structure. So you only need to store that string.